source/vibe/core/taskpool.d

```diff
@@ -10,7 +10,7 @@ module vibe.core.taskpool;
 import vibe.core.concurrency : isWeaklyIsolated;
 import vibe.core.core : exitEventLoop, logicalProcessorCount, runEventLoop, runTask, runTask_internal;
 import vibe.core.log;
-import vibe.core.sync : ManualEvent, Monitor, createSharedManualEvent, createMonitor;
+import vibe.core.sync : ManualEvent, CoreSyncMonitor = Monitor, createSharedManualEvent, createMonitor;
```

Comment:
```suggestion
import vibe.core.sync : ManualEvent, VibeSyncMonitor = Monitor, createSharedManualEvent, createMonitor;
```

Otherwise it looks like `vibe.core.sync` is publicly importing something from druntime.